doc/html/header.html

```diff
@@ -0,0 +1,78 @@
+<!-- HTML header for doxygen 1.9.6-->
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "https://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml" lang="$langISO">
+<head>
+<meta http-equiv="Content-Type" content="text/xhtml;charset=UTF-8"/>
+<meta http-equiv="X-UA-Compatible" content="IE=11"/>
+<meta name="generator" content="Doxygen $doxygenversion"/>
+<meta name="viewport" content="width=device-width, initial-scale=1"/>
+<!--BEGIN PROJECT_NAME--><title>$projectname: $title</title><!--END PROJECT_NAME-->
+<!--BEGIN !PROJECT_NAME--><title>$title</title><!--END !PROJECT_NAME-->
+<link href="$relpath^tabs.css" rel="stylesheet" type="text/css"/>
+<!--BEGIN DISABLE_INDEX-->
+  <!--BEGIN FULL_SIDEBAR-->
+<script type="text/javascript">var page_layout=1;</script>
+  <!--END FULL_SIDEBAR-->
+<!--END DISABLE_INDEX-->
+<script type="text/javascript" src="$relpath^jquery.js"></script>
+<script type="text/javascript" src="$relpath^dynsections.js"></script>
+$treeview
+$search
+$mathjax
+$darkmode
```

Comment:
This was showing up for me as a `$darkmode` text on the top of the page.

Things seem fine without it... can we remove it?

Reply:
OK, I upgraded to 1.9.8 and this was fixed, but I had to change something else to get the light mode to work properly on my system (Mac with "dark mode" by default) -- I was getting a grey/black instead of light/dark.

Specifically, I had to set `HTML_COLORSTYLE = LIGHT`

As a part of this change I took the opportunity to upgrade `CodeDocumentation.conf.in`. Do you want me to create a branch for those changes, or should I just push here.

Reply:
Great. Feel free to push into the branch. Which version did you have before? We could throw a warning if the Doxygen version is too old.